src/schemata/definitions/field_schema/timer.yml

```
@@ -1,31 +1,45 @@
 ---
 type: object
 description: |
-  Defines a field of type `timer`.
+  Defines a field of type `timer`. Timers are used to measure time or execute time-controlled actions within a Workflow Run.
+
+  Timers have two possible modes of operation: countdown and stopwatch.
+  
+  In **countdown** mode, the timer will run for a specified duration, and fire a `on_timer_complete` Trigger upon completion.
+  In **stopwatch** mode, the timer measures time until it is stopped, either manually or through a Behavior.
+  
+  Timer fields can be assigned to individual substeps by passing the field identifier to the `timer` property.
```

```suggestion
  Defines a field of type `timer` to measure time or execute time-controlled actions.

  Timers have two possible modes of operation, countdown and stopwatch. In countdown mode, the timer will run for a specified duration and fire a [Timer Complete trigger](/schemas/definitions/workflow_template/trigger_objects/timerComplete/) upon completion. In _stopwatch_ mode, the timer measures time until it is stopped, either manually or through an [action](/schemas/definitions/workflow_template/actions/).
```
Not a fan of one sentence paragraphs, and in this case it also definitely belongs together because it describes the same thing.

Where this can be used is always displayed at the bottom of each page, so I don't think we should single out certain use cases.